acbu_burning/src/lib.rs

```
@@ -17,7 +17,7 @@ mod shared {
 pub mod token_contract {
     soroban_sdk::contractimport!(
```

Clippy (1.95.0)

[error] No such file or directory (os error 2)

(error)